parquet-format-structures/pom.xml

```diff
@@ -68,9 +68,9 @@
        </plugin>
       <!-- thrift -->
       <plugin>
-        <groupId>org.apache.thrift.tools</groupId>
-        <artifactId>maven-thrift-plugin</artifactId>
-        <version>0.1.11</version>
+        <groupId>org.apache.thrift</groupId>
+        <artifactId>thrift-maven-plugin</artifactId>
+        <version>0.10.0</version>
```

Comment:
We already have the property `thrift-maven-plugin.version` in the root pom currently pointing to `0.10.0`.

Reply:
Nice catch, since it wasn't using a property before, I didn't really look for it.